git.go

```diff
@@ -121,7 +122,7 @@ func ResolveCommit(r *git.Repository, h plumbing.Hash) (*object.Commit, error) {
 	case *object.Tag:
 		return ResolveCommit(r, o.Target)
 	default:
-		log.Warn("referenced object not supported", "type", o.Type())
+		log15.Warn("referenced object not supported", "hash", h.String(), "type", o.Type())
```

Comment:
Right, the way it is structured right now, this function doesn't get context. We might want to fix in a future PR?